### ISSUE TITLE
Itemframe Placing Bug

### ISSUE THREAD

**Issue by zOIDAdaJulian** [GeyserMC/Geyser#1938]
**Describe the bug**

Chests/anvils/... bugging with Itemframes placed at the wall besides.

**To Reproduce**

1. Place a chest, anvil or other Blocks with no full BlockHitbox
2. Place an itemframe at the wall besides
3. Half of the chest is bugging

**Expected behavior**

Normally you can't place an itemframe there.

**Screenshots / Videos**

https://youtu.be/u8Mc_hsX-YI

**Server Version and Plugins**

Paper 1.16.5 
git-Paper-457

**Geyser Dump**

https://dump.geysermc.org/eGVelosZwkvptKg36W2dJQ4XJcR84S0I

**Minecraft: Bedrock Edition Version**

Windows 10 PE
Version 1.16.201

**Additional Context**

Happens on the server-inventory branch. More Infos in the Geyser-Dump

**Comment by Camotoy** [comment]
In general, this is because item frames are an entity in Java but a block in Bedrock - therefore there is no easy way to fix this.

**Comment by Camotoy** [comment]
Slightly related: https://github.com/GeyserMC/Geyser/issues/1491

**Comment by TBYT** [comment]
Yep still replicable in that java version and in java 1.19.3. This time it takes a relog for anvils/chests to go invisible and u will hit through them, especially underneath the anvil it will make the invisible anvil fall and reappear again on the block below.
